samples/abn-sample/frontend/abn.proto

```diff
@@ -0,0 +1,63 @@
+// protoc --go_out=. --go_opt=paths=source_relative     --go-grpc_out=. --go-grpc_opt=paths=source_relative     abn/grpc/abn.proto
```

Comment:
This feels like a mistake.... Shouldn't frontend authors be importing the ABn golang package (which exposes these client methods) ... instead of compiling them from scratch.

Reply:
I think there are 3 options. I chose and suggest (b).
(a) Recommend users download and generate code. I was planning to do this in the blog article. But after my Mac M1 problems I decided its a non-starter
(b) Include the generated code in the samples; users can just copy them and use them. I opted for this choice. We can document how to generate (I did this indirectly in the Dockerfile -- I actually used the Dockerfile via Github actions to get around my M1 problems). It is minimal work for users to copy the sample files and use them. Further, if they want to use another language for the frontend they won't have the expectation that we will generate and provide; they can figure it out.
(c) Provide packages. I don't want to do the work or be responsible for maintaining.